docker/docker/src/main/java/org/arquillian/cube/docker/impl/model/DockerCube.java

```diff
@@ -139,10 +155,14 @@ public void stop() throws CubeControlException {
         }
         try {
             lifecycle.fire(new BeforeStop(id));
+
+            long currentTime = System.currentTimeMillis();
             try {
                 executor.stopContainer(id);
             } catch(NotFoundException e) {
             } catch (NotModifiedException e) {}
+            this.stoppingTimeInMillis = System.currentTimeMillis() - currentTime;
+
```

Comment:
But as you said when we start adding more and more metrics we could find another kind of integrations. This is a simple one. Maybe an integration with http://metrics.dropwizard.io/3.1.0/